src/Components/AnticipationLoading/AnticipationLoading.js

```diff
@@ -1,37 +1,35 @@
-import './AnticipationLoading.scss'
-import { useState, useEffect } from 'react'
+import "./AnticipationLoading.scss";
+import { useState, useEffect } from "react";
 
 /**
  * @returns It displays timer like ( 3...2...1 ) before actually showing text to be typed
  */
 const AnticipationLoading = () => {
-    const [time, setTime] = useState(3)
+  const [time, setTime] = useState(3);
```

Comment:
Format document with Tab size of 4
don't put ; at end of lines if not required.

Reply:
Because in this diff checker, it looks like you have changed whole document.
Instead only those changes should be reflected, which only you have done.
Makes sense ?